### PR TITLE
MOTD for SRS data outage

### PR DESCRIPTION
**Pull request by dnahodil**

### CODE REVIEW

File: motd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Satellite Data Unavailable
+Access to satellite remote sensing data is unavailable due to a planned data center outage.  We appologise for any inconvenience.<br />
```
Comment (jkburges) [inline motd]: Got an expected date/time range?

**Comment by danfruehauf** [comment]
@dnahodil Just drop a file there. No need for that. Besides we cannot deploy now because @anguss00 removed the schema plugins job on jenkins...

**Comment by danfruehauf** [comment]
(@dnahodil I have put it there)

**Comment by dnahodil** [comment]
> Just drop a file there. No need for that.
The readme *specifically* told me not to edit the content of that directory. Ok for next time.

**Comment by danfruehauf** [comment]
Generally speaking, we need to explore new ways of having a MOTD. I thought something like twitter could be good. I'll open an issue for that.

**Pull request closed without merging**